docs/adr/0026-expand-subscription-platform-metadata.md

```diff
@@ -0,0 +1,81 @@
+# Expand Subscription Platform Metadata with Google Cloud Firestore
+
+- Status: proposed
+- Deciders: Ben Bangert, Barry Chen, Bianca Danforth
```

Comment:
@bbangert @biancadanforth included you since I've chatted with you about this already.